includes/AppLeave.php

```diff
@@ -75,6 +75,27 @@
 	require_once "../LocalSettings.php";
 	require_once "Globals.php";
 	
+	// validate dates
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->